packages/browser-audio-input-react/README.md

```diff
@@ -0,0 +1,71 @@
+# Browser audio input (React)
+
+React bindings for the `@speechmatics/browser-audio-input` package, letting you manage audio input devices and permissions across browsers.
```

Comment:
Maybe worth pointing out it uses useSyncExternalStore so only works for React 18+

Reply:
Yeah, the version constraint is in the `package.json` though, so if that does happen they'll get an NPM install failure I guess.

After merging I can add a follow up note to the README for the poor souls stuck on React 17